app/lib/discourse_automation/scripts/zapier_webhook.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+DiscourseAutomation::Scriptable::ZAPIER_WEBHOOK = 'zapier_webhook'
+
+DiscourseAutomation::Scriptable.add(DiscourseAutomation::Scriptable::ZAPIER_WEBHOOK) do
+  field :webhook_url, component: :text, required: true
+
+  version 1
+
+  triggerables [:user_promoted, :user_added_to_group]
+
+  script do |context, fields|
+    webhook_url = fields.dig('webhook_url', 'value')
+
+    unless /^https:\/\/hooks\.zapier\.com\/hooks\/catch\//.match?(webhook_url)
```

Comment:
Using `start_with?` instead of the regexp here feels much easier to read/understand

```suggestion
    unless webhook_url.start_with?("https://hooks.zapier.com/hooks/catch")
```